src/source_sampling.cpp

```diff
@@ -10,6 +10,8 @@ void pyne::sampling_setup_(int* mode) {
   if (sampler == NULL) {
     std::string filename ("source.h5m");
     std::string src_tag_name ("source_density");
+    std::string cell_num_tag_name ("cell_num");
```

Comment:
These are fixed/hard coded, so make part of constructor instead.

Reply:
These constructors are not only used by DAG-MCNP, but also by python when performing tests. 
However, we can't pass `string` type from DAG-MCNP input. But we need to define these `string` type file names for PyNE unit tests. 
So we have to hard code these strings between the DAG-MCNP calling function `pyne::sampling_setup_(int* mode)`  and the specific sampler for a given condition. Thus we can define these strings in PyNE unit tests.

Reply:
Now I understand... these are hard-coded only for the FORTRAN use case.  

It is possible to pass strings from FORTRAN.  Since this is called by the `source.F90` function that is also part of this pull request, maybe we should take this opportunity to move the hard-coded strings to there.

They can remain hard-coded because we don't (yet) have a way to get strings from the DAG-MCNP input file.